packages/core/chrome/core-chrome-browser-internal/src/ui/loading_indicator.tsx

```diff
@@ -76,7 +77,7 @@ export class LoadingIndicator extends React.Component<LoadingIndicatorProps, { v
       />
     ) : (
       <EuiIcon
-        type="logoElastic"
+        type={this.props.customLogo ?? 'logoElastic'}
```

Comment:
nit: I would have called the prop `logo` and if undefined set it to `logoElastic`

```ts
logo?: string = 'logoElastic',

...

type={this.props.logo}
```